packages/astro/src/assets/fonts/utils.ts

```diff
@@ -179,9 +177,59 @@ export async function generateFallbacksCSS({
 			font: fallback,
 			// TODO: support family.as
 			name: `${family} fallback: ${fallback}`,
-			metrics: (await getMetricsForFamily(fallback, null)) ?? undefined,
 		});
 	}
 
 	return { css, fallbacks };
 }
+
+/**
+ * We want to show logs related to font downloading (fresh or from cache)
+ * However if we just use the logger as is, there are too many logs, and not
+ * so useful.
+ * This log manager allows avoiding repetitive logs:
+ * - If there are many downloads started at once, only one log is shown for start and end
+ * - If a given file has already been logged, it won't show up anymore (useful in dev)
+ */
+// TODO: test
```

Comment:
nit: Not sure a test is needed for this, but if you're happy to test it, I can't stop you :)